### ISSUE TITLE
[Bug-WPF] FontFamily not working for FontAwesome

### ISSUE THREAD

**Issue by tessarolli** [xamarin/Xamarin.Forms#11789]
### Description
After updating to 4.8, Labels no longer show FontAwesome Icons on WPF Application.
All other Platforms are working as they should.

### Steps to Reproduce
```
<OnPlatform x:Key="FontAwesomeSolid" x:TypeArguments="x:String">
                <On Platform="Android">FontAwesome5_Solid.otf#Regular</On>
                <On Platform="iOS">FontAwesome5Free-Solid</On>
                <On Platform="UWP">/Assets/Fonts/FontAwesome5_Solid.otf#Font Awesome 5 Free</On>
                <On Platform="WPF">pack://application:,,,/Fonts/#Font Awesome 5 Free Solid</On>
            </OnPlatform>

 public static class FASolid
    {
public static string Eraser { get => "\uf12d"; }
}
xmlns:res="clr-namespace:myNameSpace.Resources"
<Label Text="{Static res:FASolid.Eraser}" 
                               FontFamily="{StaticResource FontAwesomeSolid}"
                        </Label>

```
### Basic Information

- Version with issue: 4.8+
- Last known good version: 4.5

### Screenshots

Working (UWP):
![image](https://user-images.githubusercontent.com/11618432/90275276-64450080-de38-11ea-810f-db8d5b4653d4.png)

Not Working (WPF):
![image](https://user-images.githubusercontent.com/11618432/90275353-89397380-de38-11ea-8470-e947747c483b.png)

### Workaround

Didn't find any, yet..

**Comment by tessarolli** [comment]
[BugReport.zip](https://github.com/xamarin/Xamarin.Forms/files/5084704/BugReport.zip)

**Comment by lindexi** [comment]
Why it can not show the right character? 

Because the font is `pack://application:,,,/Fonts/#Font Awesome 5 Free Solid, Assets/Fonts/pack://application:,,,/Fonts/.ttf#Font Awesome5Free Solid, Assets/Fonts/pack://application:,,,/Fonts/.otf#Font Awesome5Free Solid`

![](https://user-images.githubusercontent.com/16054566/91721742-d0ab5800-ebcb-11ea-96fc-8b7d642997c1.png)

**Comment by lindexi** [comment]
Why it will set this font name?

See 

https://github.com/xamarin/Xamarin.Forms/blob/fce7c48f378d5f3578b70738d7f60e78dda0c93a/Xamarin.Forms.Platform.WPF/Extensions/FontExtensions.cs#L120-L142

It will join the all the font: `var formatted = string.Join(", ", GetAllFontPossibilities(fontFamily));`

**Comment by tessarolli** [comment]
> Why it can not show the right character?
> 
> Because the font is `pack://application:,,,/Fonts/#Font Awesome 5 Free Solid, Assets/Fonts/pack://application:,,,/Fonts/.ttf#Font Awesome5Free Solid, Assets/Fonts/pack://application:,,,/Fonts/.otf#Font Awesome5Free Solid`
> 
> ![](https://user-images.githubusercontent.com/16054566/91721742-d0ab5800-ebcb-11ea-96fc-8b7d642997c1.png)

I've tried your solution, it did not help.

`<On Platform="WPF">pack://application:,,,/Fonts/#Font Awesome 5 Free Solid, Assets/Fonts/pack://application:,,,/Fonts/.ttf#Font Awesome5Free Solid, Assets/Fonts/pack://application:,,,/Fonts/.otf#Font Awesome5Free Solid</On>`
I've also tried without the Assets/ on the path.

![image](https://user-images.githubusercontent.com/11618432/91724083-24d61e00-eb73-11ea-9b53-aced5235f2c5.png)

**Comment by lindexi** [comment]
@tessarolli I try to fix this in https://github.com/xamarin/Xamarin.Forms/pull/11977

**Comment by DennisWelu** [comment]
@samhouts @PureWeen Looks like this is in and out of the priority list for upcoming releases. What's the thought on a future fix? Looks like @lindexi has a PR in and just pending review. If the fix doesn't make it into a 4.8.x SR, as it appears, it will force me to update everything in the solution to 5.x when that is GA (due to other changes like MasterDetailPage name change etc.). Kinda leaves 4.x WPF solutions hanging.

**Comment by DennisWelu** [comment]
Can someone explain the relationship the [5.0.1 project](https://github.com/xamarin/Xamarin.Forms/projects/83) has to the 5.0.0 SR process? This fix has been Done for quite some time in the 5.0.1 project and hasn't made it to a SR that I'm aware. When is 5.0.1 targeted for?
